.github/workflows/release-internal.yml

```diff
@@ -51,14 +43,14 @@ env:
 
 jobs:
   invoke-reusable-workflow:
-    name: Build and Release Internally
+    name: "Build and Release Internally"
 
-    uses: VersusFacit/dbt-release/.github/workflows/internal-archive-release.yml@main
+    uses: "dbt-labs/dbt-release/.github/workflows/internal-archive-release.yml@mp/finish_internal_workflow"
```

Comment:
Going to change this once `main` has all the changes needed to support all the other workflows. Since this PR's workflow can be invoked using the branch reference, I'll merge at the time I can revert to main (unless there is an alternate path desired)